plantuml-parser-core/src/test/java/com/shuzijun/plantumlparser/core/TestRecordAndDefaultConstructor.java

```diff
@@ -0,0 +1,136 @@
+package com.shuzijun.plantumlparser.core;
+
+import com.github.javaparser.ParserConfiguration;
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.PrintStream;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+import java.util.Scanner;
+
+public class TestRecordAndDefaultConstructor {
```

Comment:
I think it would be better to give it a more general name, such as `UmlGenerationTester`.